app/ui-react/syndesis/src/modules/integrations/components/editor/dataMapper/utils.ts

```diff
@@ -184,10 +180,8 @@ export function getOutputDocument(
   const outputDocuments = subsequentSteps
     .map(s => stepToProps(s.step, false, true, s.index))
     .filter(s => s !== false && s.id !== stepId) as IDocumentWithShape[];
-
   if (outputDocuments.length === 0) {
     throw new Error('output document shape kind not supported');
   }
-
-  return outputDocuments[0];
+  return outputDocuments[0] || {};
```

Comment:
Replying here to your comment above. Is an empty object an ok document for atlasmap then?

Reply:
oh crap, I should take that out, it's a leftover, tx for spotting.